sympy/physics/pring.py

```diff
@@ -7,9 +7,12 @@ def wavefunction(n, x):
     """
     Returns the wavefunction for particle on ring.
 
-    n is the quantum number, x is the angle,
-    here n can be positive as well as negative
-    which can be used to describe the direction of motion of particle
+    Explanation
+    ===========
+
+    ``n`` is the quantum number, ``x`` is the angle,
```

Comment:
Would be good to add these as Parameters? Is this function not in public API?

Reply:
Added these as parameters.